wallet/bindings/java/lib/native/src/lib.rs

```diff
@@ -90,6 +101,13 @@ pub extern "system" fn Java_org_iota_api_NativeApi_createMessageHandler(
 ) {
     env_assert!(env, ());
 
+    #[cfg(target_os = "android")]
+    android_logger::init_once(
+        Config::default()
+            .with_tag("lib_wallet")
+            .with_max_level(LevelFilter::Trace),
```

Comment:
Just a heads up, Trace in an insane amount of logs...

Reply:
Yes you're right, as said before, logcat is too verbose and is not needed now

Reply:
I've disabled it but keeping the code so we can debug for potential future issues